### PR TITLE
Add GraphQL endpoint option for Playground

### PR DESCRIPTION
**Pull request by tomphilbin**
Adds ability to specify an endpoint for playground:

```
app.UsePlayground(new PlaygroundOptions() 
{
    GraphQLEndpoint = "https://hotchocolate.io/graphql"
});

app.UseVoyager(new VoyagerOptions() 
{
    GraphQLEndpoint = "https://hotchocolate.io/graphql"
});
```

Addresses #906

### CODE REVIEW

File: src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs [hunk 1]
```diff
@@ -17,6 +18,27 @@ public PlaygroundOptions()
...
 
+        /// <summary>
+        /// The endpoint of the GraphQL service.
+        /// </summary>
+        public string? GraphQLEndpoint
+        {
+            get => _graphQLEndpoint?.AbsoluteUri;
+            set
+            {
+                try
```
Comment (michaelstaib) [inline src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs]: What is the original exception? I think the exception that Uri throws might be enough.

File: src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs [hunk 2]
```diff
@@ -17,6 +18,27 @@ public PlaygroundOptions()
             _path = _defaultPath = new PathString("/playground");
         }
 
+        /// <summary>
+        /// The endpoint of the GraphQL service.
+        /// </summary>
+        public string? GraphQLEndpoint
```
Comment (michaelstaib) [inline src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs]: Should we even make this a URI so there cannot be an exception? @PascalSenn thoughts?

**Comment by michaelstaib** [comment]
@tomphilbin can you also do this for the voyager middleware?

File: src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs (hunk 1 above)
Comment (tomphilbin) [reply src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs]: You get `ArgumentNull` or `UriFormatException` - both quite informative, will update :)

**Comment by michaelstaib** [comment]
@tomphilbin also when we are through we need a second PR that targets branch version_10_0_0_master so that we can get this in as a hot fix for version 10. Version 10 and 11 can no longer merge.

**Comment by michaelstaib** [comment]
@tomphilbin great start thanks for your help

File: src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs (hunk 2 above)
Comment (PascalSenn) [reply src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs]: I generally do not like overring getter and setter. 
So asking me, yes we should

Comment (tomphilbin) [reply src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs]: Ha, same here! Was aiming to be consistent with the rest of the class 😄. Will update.

Comment (michaelstaib) [reply src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs]: So lets make the property a Uri?

**Comment by michaelstaib** [comment]
@tomphilbin once the builds are through we will merge. Welcome as a Contributor to Hot Chocolate!

**Comment by michaelstaib** [comment]
@tomphilbin can you create another PR to these changes onto 10.4.1?

**Comment by michaelstaib** [comment]
The sonar build needs to update the sonar tokens.

**Pull request merged**